packages/common/src/sandbox/modules.ts

```diff
@@ -249,34 +250,36 @@ export const isMainModule = (
   return path.replace('/', '') === entry;
 };
 
-export const findMainModule = (
-  modules: Module[],
-  directories: Directory[],
-  entry: string = 'index.js',
-  template?: TemplateType
-) => {
-  try {
-    // first attempt: try loading the entry file if it exists
-    const module = resolveModule(entry, modules, directories);
-    return module;
-  } catch (e) {
-    try {
-      // second attempt: try loading the first file that exists from
-      // the list of possible defaults in the template defination
-      const templateDefinition = getTemplateDefinition(template);
+export const findMainModule = (sandbox?: Sandbox) => {
+  const resolve = resolveModuleWrapped(sandbox);
 
-      const defaultOpenedFiles = templateDefinition.getDefaultOpenedFiles({});
+  // first attempt: try loading the entry file if it exists
+  const entryModule = resolve(sandbox.entry);
+  if (entryModule) return entryModule;
```

Comment:
Put curly braces around body to make this less error-prone.

Reply:
Like this:?

```js
if (entryModule) {
  return entryModule;
}
```

Reply:
Yes that's what I ment indeed 🙂

Reply:
Done